packages/vitest/src/node/pools/child.ts

```diff
@@ -70,20 +72,20 @@ export function createChildProcessPool(ctx: Vitest, { execArgv, env, forksPath }
 
     env,
     execArgv: [
-      ...ctx.config.poolOptions?.forks?.execArgv ?? [],
+      ...poolOptions.execArgv ?? [],
       ...execArgv,
     ],
 
     terminateTimeout: ctx.config.teardownTimeout,
+    concurrentTasksPerWorker: 1,
```

Comment:
Priscina mentions that it is already 1 by default anyway, but for some reasons, we also set it conditionally. I haven't checked tinypool implementation, but the documentation doesn't mention that this value differs.